headers/types/dungeon_mode/dungeon_mode_common.h

```diff
@@ -53,9 +53,10 @@ struct move {
     // Consume 4 PP this turn. No effect unless f_consume_pp is set, overrides f_consume_2_pp
     bool f_consume_4_pp : 1;
     uint16_t flags2_unk7 : 1;
-    uint16_t flags2_unk8 : 1;
+    bool f_multitalent_iq_pp_boost : 1; // The IQ skill Multitalent is actively boosting move PP
```

Comment:
This name makes it sound like multitalent is boosting IQ. Can we just call it `f_multitalent_pp_boost`? I think "multitalent" is distinct enough on its own.